src/Idephix/Extension/Slack/Slack.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace Idephix\Extension\Slack;
+
+use Idephix\IdephixInterface;
+use Idephix\Extension\IdephixAwareInterface;
+
+/**
+ * Description of Slack wrapper
+ *
+ * @author dymissy
+ */
+class Slack implements IdephixAwareInterface
+{
+    /**
+     * @var \Idephix\IdephixInterface
+     */
+    private $idx;
+
+    private $settings;
+
+    public function __construct($args = array())
```

Comment:
:+1: 

Consistency is important to maintain.

@micheleorselli: we could remove the constructor of the class and inject all needed parameters to ```sendToSlack```method. Is this your suggestion ?